pkg/core/models.go

```diff
@@ -456,3 +454,29 @@ type TestExecutionArgs struct {
 	FrameWorkVersion  int
 	CWD               string
 }
+
+// YMLParsingRequestMessage defines yml parsing request received from TAS server
+type YMLParsingRequestMessage struct {
+	GitProvider string    `json:"gitProvider"`
+	CommitID    string    `json:"commitID"`
+	Event       EventType `json:"eventType"`
+	RepoSlug    string    `json:"repoSlug"`
+	TasFileName string    `json:"tasFilePath"`
+	LicenseTier Tier      `json:"license_tier"`
+	OrgID       string    `json:"orgID"`
+	BuildID     string    `json:"buildID"`
+}
+
+// TASConfigDownloaderOutput repersent output return by by tasconfig downloader
+type TASConfigDownloaderOutput struct {
+	Version   int         `json:"version"`
+	TasConfig interface{} `json:"tasConfig"`
```

Comment:
```suggestion
	TASConfig interface{} `json:"tasConfig"`
```

Reply:
Won't `TASConfig` have a version in itself?

Reply:
Both way is fine , in current approach , user can just check the version and parse YML accordingly
In second version , he has to parse YML twice, first to check version and then to get exact TAS config
I think first approach is simpler but open to change